src/st_stuff.c

```diff
@@ -1306,16 +1302,17 @@ static int StatusBarBufferHeight(void)
 {
   int i;
   int st_height = ST_HEIGHT;
+  patch_t *const patch = W_CacheLumpName("brdr_b", PU_CACHE);
+
+  if (patch && patch->height > st_height)
```

Comment:
While I'm here, is there any reason to actually check the pointer first? Can we assume it's always valid? I should just remove all of these I think.

Reply:
https://github.com/fabiangreffrath/woof/blob/a19eaca101fc137a5b3c3a8a9dca9a8f08b5b96c/src/w_wad.c#L454-L467

https://github.com/fabiangreffrath/woof/blob/a19eaca101fc137a5b3c3a8a9dca9a8f08b5b96c/src/w_wad.c#L396

Initialized to zero. I should just remove all of these pointless checks.

Reply:
I guess we can remove them, there will be an `I_Error()` if the patch is not found or cannot be read.

Reply:
On second thought, better safe than sorry, because the various static `patch_t` pointers could in theory be uninitialized.